Example/Tests/ImagePickerViewControllerTests.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  ImagePickerViewControllerTests.swift
+//  GiniVision_Tests
+//
+//  Created by Enrique del Pozo Gómez on 2/26/18.
+//  Copyright © 2018 Gini GmbH. All rights reserved.
+//
+
+import XCTest
+@testable import GiniVision
+
+final class ImagePickerViewControllerTests: XCTestCase {
+    
+    let galleryManager = GalleryManagerMock()
+    lazy var currentAlbum = self.galleryManager.albums[1]
+    
+    lazy var vc = ImagePickerViewController(album: self.currentAlbum,
+                                            galleryManager: GalleryManagerMock(),
+                                            giniConfiguration: GiniConfiguration.sharedConfiguration)
+    
+    override func setUp() {
+        super.setUp()
+        _ = vc.view
+    }
+    
+    func testViewControllerTitle() {
+        let title = currentAlbum.title
+        XCTAssertEqual(title, vc.title, "view controller title should match the album title")
+    }
+    
+    func testNumberOfSections() {
+        XCTAssertEqual(vc.collectionView.numberOfSections, 1, "There should be only one section")
```

Comment:
Either have the assertion descriptions start with a capital letter or a lowercase one. But be consistent.

Reply:
You are right 👍